### PR TITLE
update to rbush 2.0.1

### LINKED ISSUE #4691: Update rbush version
RBush made a change recently that offers significant performance increases in many cases, but does involve an API change:

https://github.com/mourner/rbush/issues/44

### PR DESCRIPTION
**Pull request by bryevdv**
- [x] issues: fixes #4691

### CODE REVIEW

**Comment by bryevdv** [comment]
I'd like to take this opportunity to add some tests around this, but @mattpap your eyes on this would be appreciated.

**Comment by bryevdv** [comment]
Actually, the `bbox` and auto-ranging tests are passing. Adding tests to really test hit-testing/selection is a whole nother ordeal, I'd rather make a concerted effort a that at a different time.

**Comment by bryevdv** [comment]
ping @bokeh/dev I'd like to merge this Tuesday

File: bokehjs/src/coffee/core/util/bbox.coffee [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
+  r = {}
+  r.minX = Math.min(a.minX, b.minX)
+  r.maxX = Math.max(a.maxX, b.maxX)
+  r.minY = Math.min(a.minY, b.minY)
+  r.maxY = Math.max(a.maxY, b.maxY)
+  return r
 
 module.exports =
-  empty: empty
+  EMPTY: {minX: Infinity, minY: Infinity, maxX: -Infinity, maxY:-Infinity}
```
Comment (mattpap) [inline bokehjs/src/coffee/core/util/bbox.coffee]: This should be a function. Otherwise you will get this modified in-place sooner or later.

Comment (bryevdv) [inline bokehjs/src/coffee/core/util/bbox.coffee]: Yes, that's safer, I will make the change.

**Comment by bryevdv** [comment]
@mattpap I've made that change, please let me know when your review is done so that I can merge.

**Comment by mattpap** [comment]
LGTM

**Pull request merged**